src/redux/actions/fetchArticles.js

```diff
@@ -0,0 +1,57 @@
+import axios from "axios";
+import {
+  START_ARTICLE_FETCH,
+  ARTICLE_FETCH_SUCCESS,
+  ARTICLE_FETCH_ERROR
+} from "../action_types";
+import config from "../../config";
+
+const url = `${config.BASE_URL}/articles/`;
+
+/**
+ * Action for start article fetch
+ *
+ * @return {object} type
+ */
+export const startArticleFetch = () => ({
```

Comment:
Why import `startFetch()` in all the previous files but in this instance create an entire action creator?

Reply:
Fixed